src/Stratis.Bitcoin.Features.Miner/Controllers/MinerController.cs

```diff
@@ -114,5 +113,32 @@ public IActionResult StartStaking([FromBody]StartStakingRequest request)
                 return ErrorHelpers.BuildErrorResponse(HttpStatusCode.BadRequest, e.Message, e.ToString());
             }
         }
+
+        /// <summary>
+        /// Stop staking.
+        /// </summary>
+        /// <returns>An OKResult object that produces a status code 200 HTTP response.</returns>
+        [Route("stopstaking")]
+        [HttpPost]
+        public IActionResult StopStaking()
+        {
+            try
+            {
+                if (!this.ModelState.IsValid)
```

Comment:
Do you need this if there is no model?

Reply:
You tell me, I am not sure.

Reply:
Yes, you don't need to check the ModelState here as you didn't post anything to the controller method.

Reply:
Yes I think the modelstate.isvalid can be removed on any controller method that does not have any parameters. I think you can remove it for GetStakingInfo() as well.